java/main/com/google/privacy/differentialprivacy/Count.java

```diff
@@ -97,14 +98,38 @@ public long computeResult() {
     }
 
     resultReturned = true;
-    return params
-        .noise()
-        .addNoise(
-            rawCount,
-            params.maxPartitionsContributed(),
-            params.maxContributionsPerPartition(),
-            params.epsilon(),
-            params.delta());
+    noisedCount = params
+            .noise()
+            .addNoise(
+                    rawCount,
+                    params.maxPartitionsContributed(),
+                    params.maxContributionsPerPartition(),
+                    params.epsilon(),
+                    params.delta());
+    return noisedCount;
+  }
+
+  // ComputeConfidenceInterval computes a confidence interval with integer bounds that contains the true count with
+  // a probability greater or equal to 1 - alpha using the noised count computed by computeResult().
+  public ConfidenceInterval computeConfidenceInterval(double alpha) {
+    if (!resultReturned) {
+      throw new IllegalStateException("Noised count must be computed before calling this function.");
+    }
+    ConfidenceInterval confInt =
+            params
+                    .noise()
+                    .computeConfidenceInterval(
+                            noisedCount,
+                            params.maxPartitionsContributed(),
+                            params.maxContributionsPerPartition(),
+                            params.epsilon(),
+                            params.delta(),
+                            alpha);
+    confInt =
+            ConfidenceInterval.create(
+                    Math.round(Math.max(0, confInt.lowerBound())),
+                    Math.round(Math.max(0, confInt.upperBound())));
+    return confInt;
```

Comment:
just return ConfidenceInterval.create(
                    Math.round(Math.max(0, confInt.lowerBound())),
                    Math.round(Math.max(0, confInt.upperBound())));

without assigning it to confInt, it is unnessesary